app/views/staff/activity_forms/parent.html.haml

```diff
@@ -0,0 +1,9 @@
+= render layout: "wrapper" do |f|
+  - scoped_parent_activities = scoped_parent_activities(activity: f.object, user: current_user)
+  = f.hidden_field :parent, value: nil
+  = f.govuk_collection_radio_buttons :parent,
+    scoped_parent_activities,
+    :id,
+    ->(activity){ ActivityPresenter.new(activity).display_title },
+    legend: { tag: 'h1', size: 'xl', text: I18n.t("form.legend.activity.parent", parent_level: f.object.parent_level, level: f.object.level) },
```

Comment:
This is another place where 'Third-party project' bites us! setting `level` to something like

`level: t("page_content.activity.level.#{f.object.level}")` is the approach we've used elsewhere.

Reply:
I agree! Will switch to internationalisation.